SimpleKeychain/A0SimpleKeychain.h

```diff
@@ -141,6 +145,14 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property (assign, nonatomic) BOOL useAccessControl;
 
+
+/**
+*  LocalAuthenticationContext used to access items. Default value is a new LAContext object
+*/
+#if (defined __IPHONE_8_0 || defined __MAC_10_12) && !defined __TVOS_9_0 && !defined __WATCHOS_1_0
```

Comment:
Not sure on the logic here to also check for the library ! to be a specific version, would be better I think to use MIN and you only need to specify for iOS and Mac OS e.g.

```
#if __IPHONE_OS_VERSION_MIN_ALLOWED >= 90000
```

Reply:
You are right. Would you agree to create a Macro in A0SimpleKeychain.h with the following definition?

```
#define A0LocalAuthenticationCapable __IPHONE_OS_VERSION_MIN_ALLOWED >= 80000 || __MAC_OS_X_VERSION_MIN_REQUIRED >= 101200
```
So when necessary, we can use it like this

```
#if A0LocalAuthenticationCapable
   ...
#endif
```

Thanks!